### PR TITLE
fix(ci): sync delivery 2210

### PR DESCRIPTION
**Pull request by tuntoja**
## Description

**PLEASE MAKE SURE THAT THE BRANCH PR INCLUDES JIRA TICKET ID** (_for centreon-internal_)

Please include a short resume of the changes and what is the purpose of PR. Any relevant information should be added to help:
* **QA Team** (Quality Assurance) with tests.
* **reviewers** to understand what are the stakes of the pull request.

**Fixes** # (issue)

## Type of change

- [x] Patch fixing an issue (non-breaking change)
- [ ] New functionality (non-breaking change)
- [ ] Breaking change (patch or feature) that might cause side effects breaking part of the Software

## Target serie

- [ ] 21.10.x
- [ ] 22.04.x
- [x] 22.10.x
- [ ] 23.04.x (master)

<h2> How this pull request can be tested ? </h2>

Please describe the **procedure** to verify that the goal of the PR is matched. Provide clear instructions so that it can be **correctly tested**.

Any **relevant details** of the configuration to perform the test should be added.

#### Community contributors & Centreon team

- [ ] I have followed the **coding style guidelines** provided by Centreon
- [ ] I have commented my code, especially new **classes**, **functions** or any **legacy code** modified. (***docblock***)
- [ ] I have commented my code, especially **hard-to-understand areas** of the PR.
- [ ] I have **rebased** my development branch on the base branch (master, maintenance).

### CODE REVIEW

**Review by paloth — APPROVED** [review]

File: .github/workflows/create-jira-version-awie.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+name: release-web
```
Comment (bdauria) [inline .github/workflows/create-jira-version-awie.yml:2]: change this name (applies to other wf files too)

Comment (tuntoja) [reply .github/workflows/create-jira-version-awie.yml:2]: @bdauria  would "oss" be better (if we consider this is at a release level): release-oss
or "module_name" better (if we consider them individually): release-<module_name>

Comment (tuntoja) [reply .github/workflows/create-jira-version-awie.yml:2]: release-<module_name> was selected as a solution after discussion with @bdauria

Comment (tuntoja) [reply .github/workflows/create-jira-version-awie.yml]: suggestion was applied

**Review by bdauria — APPROVED** [review]

**Pull request merged**